Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -1,145 +1,114 @@
\ No newline at end of file
+Assignment Part One: Object-Property Pair contradictions.
+
+"PROVIDE OWL-2 DIRECT SEMANTIC BASED EXPLANATIONS
+FOR UNSATISFIABILITY AND INCONSISTENCY RESULTS
+STEMMING FROM ROLE CONSTRAINT PAIRS."
+
+1.) FUNCTIONAL COMBINED WITH TRANSITIVE BECOMES UNDECIDEABLE
+
+Functional object property denotes cardinality of (1) such relationship between 2 instances.
+For example, John (x) is connected to Mary (y) by object-property "has father" is unique.
```

Comment:
Switch Mary with Steve or vis a versa. For every (y) related to (x) there can only be one (x) who hold this property relation

Reply:
Please see "Initial Work..." file that does not have messed up code of the left hand side for commenting.